site/app/templates/forum/ShowForumThreads.twig

```diff
@@ -89,7 +89,9 @@
             <div id="thread_list" class="col-3" data-prev_page="{{ prev_page }}" data-next_page="{{ next_page }}">
                 <i class="fas fa-spinner fa-spin fa-2x fa-fw fill-available" style="color:gray;display: none;" aria-hidden="true"></i>
                 <i class="fas fa-caret-up fa-2x fa-fw fill-available" style="color:gray; {{ arrowup_visibility }}" aria-hidden="true"></i>
-
+                <script>
+                    const isFullPage = false;
+                </script>
```

Comment:
This definitely feels like the wrong solution to me.  Seemingly random variables floating around all over the place, completely detached from the places they are used, quickly leads to major technical debt issues.  In fact, this kind of thing is precisely why the forum is so difficult to work on.  While I have some big-picture ideas for how to resolve this kind of problem across the codebase eventually, I don't have a good alternative solution to propose for this specific case.  @shailpatels or @yanliw123, any thoughts here?  I don't want to block this PR without having specific suggested improvements, but I also think it would be worthwhile to do a bit more thinking here.